snippets/base/views.py

```diff
@@ -47,13 +47,14 @@ def template_name(self):
 
 
 def fetch_snippets(request, **kwargs):
-    if settings.USE_PREGEN_BUNDLES:
+    if settings.USE_PREGEN_BUNDLES and kwargs['startpage_version'] == 6:
```

Comment:
Only version 6 or `>= 6`?

Reply:
Will keep '6' for now. Who knows what `7` will look like

Reply:
Fair.